workflow_scripts/test_new_models.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+from pathlib import Path
+import subprocess
+from utils import test_utils
+
+
+def main():
+  parser = argparse.ArgumentParser(description='Test settings')
+  # default all: test by both onnx and onnxruntime
+  # if target is specified, only test by the specified one
+  parser.add_argument('--target', required=False, default='all', type=str, 
```

Comment:
Duplicate -- `--target` arguments. Not sure whether these two need to be merged together